src/core/p5.Renderer2D.js

```diff
@@ -1128,20 +1128,12 @@ p5.Renderer2D.prototype.scale = function(x, y) {
 };
 
 p5.Renderer2D.prototype.shearX = function(angle) {
```

Comment:
hmm, maybe worth a comment that unlike it's p5.prototype counterpart, this method expects radians only? same for shearY. I see that this is already true for rotate, so it makes sense to handle this the same way—maybe there's room to add a note about this general pattern somewhere?

Reply:
yeah, i think the assumption now is that the renderers _only_ deal in radians, and the angleMode stuff is all contained within the p5 instance. i'll put a comment to that affect near the `_angleMode` declaration.